code/modules/projectiles/projectile/bullets/sniper.dm

```diff
@@ -7,12 +7,13 @@
 	knockdown = 100
 	dismemberment = 50
 	armour_penetration = 60
+	var/obj_damage_mod = 1.1
```

Comment:
What is this change for?

Reply:
snowflaked gun (changing hardcoded object damage to a modifier). Will look again at it in another pr.